source/mechanics/fiber_network.cc

```diff
@@ -0,0 +1,62 @@
+#include <fiddle/mechanics/fiber_network.h>
+#include <deal.II/grid/tria.h>
+#include <deal.II/base/array_view.h>
+#include <deal.II/base/table.h>

```

Comment:
The header inclusions here are not in the correct order - can you install clang-format and run it on all the new files you added? That will also fix some indentation issues.